server/chat-plugins/uno.ts

```diff
@@ -435,7 +401,7 @@ class UnoGame extends Rooms.RoomGame {
 		// update the unoId here, so when the display is sent to the player when the play is made
 		if (player.hand.length === 1) {
 			this.awaitUno = player.id;
-			this.unoId = Math.floor(Math.random() * 100).toString();
+			this.unoId = toID(Math.floor(Math.random() * 100).toString());
```

Comment:
```suggestion
			this.unoId = Math.floor(Math.random() * 100).toString() as ID;
```

Reply:
im not sure what difference this makes

Reply:
You're changing code that's used in run-time to appease TypeScript, which only runs at compile-time. There's no reason to do that unless it's the only solution.